log_outgoing_requests/handlers.py

```diff
@@ -100,5 +102,53 @@ def emit(self, record: AnyLogRecord):
 
         OutgoingRequestsLog.objects.create(**kwargs)
 
+    def log_exception(self, record, config):
```

Comment:
the original implementation already had some support for exceptions via

```
trace = traceback.format_exc() if record.exc_info else ""
```

This code duplication is a smell to me that it should just be a single method/function to do the logging. There is too much risk with header/credential scrubbing etc. to maintain parallel implementations.

I'd rather see the code made more robust in that the response can be optional. Requests itself does this too - their exception class takes an optional `response` kwarg which may be `None`. Perhaps that should also be the mechanism that powers this - the exception itself being a `requests.RequestException` gives you access to the response (and request? not sure).